dev/collect.py

```diff
@@ -154,19 +203,19 @@ def scrape_job_preview(self, job_pages=[]):
                 "City": [location[0] for location in locations],
                 "State": [location[1] for location in locations],
                 "ZipCode": [location[2] for location in locations],
-                "WorkType": work_types
-                # "TimeStatus":
-                # "MinSalary":
-                # "MaxSalary":
-                # "SalaryEstimated":
-                # "QuickApply":
-                # "UrgentHire":
+                "WorkType": work_types,
+                # "WorkTimeStatus": self._collect_worktime_status(job_cards)
+                "MinSalary": [salary[0] for salary in salaries],
+                "MaxSalary": [salary[1] for salary in salaries],
+                "SalaryEstimated": estimated,
+                "EasyApply": easy_apply,
+                "UrgentHire": urgent_hire,
```

Comment:
Double check the commas in dictionaries when there are incompleted key:value pairs. The last working element (which includes all of the non-commented ones) should not have a comma.